index.html

```diff
@@ -0,0 +1,166 @@
+          <em>Expo tuiteratura 2022</em> es una iniciativa de Ana Baños,
+          <a href="https://twitter.com/sonabana_art">@sonabana_art</a>, arropada
+          con entusiasmo por 200 noveles hilanderos, todos jóvenes
+          universitarios, estudiantes de 4to, 7mo y 9no semestre de la facultad
+          de Psicología - UANL.
+        </p>
+        <h4 class="event-dates">25 de abril al<br />15 de mayo de 2022</h4>
+        <h4 class="event-organizer">
+          <a href="mailto:sonaban.art@gmail.com">sonaban.art@gmail.com</a>,
+          <a href="https://twitter.com/sonabana_art">@sonabana_art</a>
+        </h4>
+      </section>
+
+      <section id="program" class="section program">
+        <h2 class="section-title">Programa</h2>
+        <div class="separator"></div>
+        <div class="program-container">
+          <div class="program-item">
+            <div class="program-icon">
+              <img src="resources/images/program_icon_01.png" alt="icon-1" />
+            </div>
+            <div class="program-title">Conferencia</div>
+            <div class="program-description">
+              Excucha a los expositores de varios países compartiendo sus
+              experiencias.
+            </div>
+          </div>
+          <div class="program-item">
+            <div class="program-icon">
+              <img src="resources/images/program_icon_02.png" alt="icon-1" />
+            </div>
+            <div class="program-title">Exhibición</div>
+            <div class="program-description">
+              Aprecia la creación de los hilanderos.
+            </div>
+          </div>
+          <div class="program-item">
+            <div class="program-icon">
+              <img src="resources/images/program_icon_03.png" alt="icon-1" />
+            </div>
+            <div class="program-title">Foro</div>
+            <div class="program-description">
+              Comparte tus ideas y opiniones con los participantes.
+            </div>
+          </div>
+          <div class="program-item">
+            <div class="program-icon">
+              <img src="resources/images/program_icon_04.png" alt="icon-1" />
+            </div>
+            <div class="program-title">Taller</div>
+            <div class="program-description">
+              Intenta crear tus propias obras de tuiteratura, para no quedarte
+              como observador.
+            </div>
+          </div>
+          <div class="program-item">
+            <div class="program-icon">
+              <img src="resources/images/program_icon_05.png" alt="icon-1" />
+            </div>
+            <div class="program-title">Networking</div>
+            <div class="program-description">
+              Participa formando una red social de persona interesadas en la
+              creación de tuiteratura.
+            </div>
+          </div>
+        </div>
+        <h3 class="program-join">Regístrate en EXPO tuiteratura 2022</h3>
```

Comment:
- [ ] In the desktop version this should be a link instead of a header, please fix that.
- [ ] Please make sure to add the hover effect on the individual card so that it has borders.

### Behance Design
![image](https://user-images.githubusercontent.com/59416136/163452416-afa80ed8-aa6d-459d-a268-9a89a9b69584.png)


### Your Implementation

![image](https://user-images.githubusercontent.com/59416136/163452263-f05d16f4-21cd-4684-8c44-f6a7f07fd8b6.png)